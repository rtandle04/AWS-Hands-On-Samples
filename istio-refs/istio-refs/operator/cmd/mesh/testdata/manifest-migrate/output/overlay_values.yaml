apiVersion: install.istio.io/v1alpha1
kind: IstioOperator
metadata:
  creationTimestamp: null
spec:
  gateways:
    components:
      ingressGateway:
        enabled: true
        k8s:
          nodeSelector:
            scalers.istio: dedicated
          tolerations:
          - effect: NoSchedule
            key: scalers.istio
            operator: Equal
            value: dedicated
    enabled: true
  policy:
    components:
      policy:
        enabled: false
    enabled: false
  telemetry:
    components:
      telemetry:
        enabled: false
    enabled: false
  trafficManagement:
    components:
      pilot:
        enabled: true
        k8s:
          nodeSelector:
            scalers.istio: dedicated
          tolerations:
          - effect: NoSchedule
            key: scalers.istio
            operator: Equal
            value: dedicated
    enabled: true
  values:
    gateways:
      istio-ingressgateway:
        autoscaleMax: 20
        autoscaleMin: 20
        podAntiAffinityTermLabelSelector:
        - key: istio
          operator: In
          topologyKey: kubernetes.io/hostname
          values: ingressgateway
    pilot:
      autoscaleMax: 20
      autoscaleMin: 20
      podAntiAffinityTermLabelSelector:
      - key: istio
        operator: In
        topologyKey: kubernetes.io/hostname
        values: pilot

