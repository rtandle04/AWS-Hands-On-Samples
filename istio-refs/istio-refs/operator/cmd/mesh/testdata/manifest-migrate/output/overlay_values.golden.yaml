apiVersion: install.istio.io/v1alpha1
kind: IstioOperator
metadata:
  creationTimestamp: null
spec:
  components:
    ingressGateways:
    - enabled: true
      k8s:
        nodeSelector:
          scalers.istio: dedicated
        tolerations:
        - effect: NoSchedule
          key: scalers.istio
          operator: Equal
          value: dedicated
      name: istio-ingressgateway
    pilot:
      k8s:
        nodeSelector:
          scalers.istio: dedicated
        tolerations:
        - effect: NoSchedule
          key: scalers.istio
          operator: Equal
          value: dedicated
    policy:
      enabled: false
    telemetry:
      enabled: false
  values:
    gateways:
      istio-ingressgateway:
        autoscaleMax: 20
        autoscaleMin: 20
        podAntiAffinityTermLabelSelector:
        - key: istio
          operator: In
          topologyKey: kubernetes.io/hostname
          values: ingressgateway
    pilot:
      autoscaleMax: 20
      autoscaleMin: 20
      podAntiAffinityTermLabelSelector:
      - key: istio
        operator: In
        topologyKey: kubernetes.io/hostname
        values: pilot

